README.md

```diff
@@ -0,0 +1,18 @@
+# tup-services
+
+Services API and clients for TACC User Portal functions
+
+## Client
+
+The client has been built using [Vite](https://vitejs.dev/guide/#trying-vite-online) and [React (TypeScript)](https://reactjs.org/). It was configured with [NodeJS v16.15.0 (LTS)](https://nodejs.org/en/). To run it, you may use the following shell commands from within the [`tup-ui`](./tup-ui) directory:
+
+```
+npm install
```

Comment:
```suggestion
npm ci
```

Using `npm ci`[^2] will use `pacakge-lock.json`'s explicit versions[^1], so PRs don't include large hidden `package-lock.json` diffs that change versions under our nose.

[^1]: (instead of the flexible versions in `package.json`)
[^2]: (like Core-CMS, Core-Portal, and Core-Styles' READMEs)